main/src/main/java/cgeo/geocaching/ui/dialog/SimpleDialog.java

```diff
@@ -218,6 +219,22 @@ private void applyCommons(final AlertDialog.Builder builder) {
         }
     }
 
+    private void applyCommonsForSelectionDialogs(final AlertDialog.Builder builder) {
+        // By default message is not supported if a listview is shown, therefore we need to build the dialog ourselves and set it as header
+        if (message != null || title != null) {
```

Comment:
Wouldn't it be enough to apply the custom header if message != null? Setting a custom title (without message) is also supported for selection dialog w/o custom header.

Reply:
Yes in principle, although this would slightly increase complexity cause we would need to set the title in two different ways depending on whether message != null.

Do you have a preference?

Reply:
I would call applyCommons in case message is null.

Anyway, current implementation is fine too.

Reply:
Ok, will change it...